docs/index.md

```diff
@@ -20,6 +20,9 @@ To install the Shorebird command line interface (CLI):
 
 ```bash
 curl --proto '=https' --tlsv1.2 https://raw.githubusercontent.com/shorebirdtech/install/main/install.sh -sSf | sh
+
+# or alternative to Linux, if the command above not working. Use bash instead of sh.
+curl --proto '=https' --tlsv1.2 https://raw.githubusercontent.com/shorebirdtech/install/main/install.sh -sSf | bash
```

Comment:
Thanks for the patch!

I think we just change the above line to use | bash instead of | sh.

Reply:
great, i assume, it is also work on mac. isn't it? If so then I can remove this PR. love your works btw.

Reply:
Can you share the logs of the failure of the first command on your machine so we can better understand what needs fixing here?

is it that `sh` doesn't exist in your path?   Or is it that `sh` has the wrong behavior?  Piping to bash seems reasonable, so a change which changed the first line to use `bash` instead of `sh` seems like a good one and we would welcome such a patch.

Reply:
I think the issue related to this.
[stackoverflow](https://stackoverflow.com/a/1878999)

![grafik](https://user-images.githubusercontent.com/116447203/235239644-72463419-ebd9-4e1a-9e91-d861d4f819d9.png)

Some output from the command.
```
user@workstation:~$ curl --proto '=https' --tlsv1.2 https://raw.githubusercontent.com/shorebirdtech/install/main/install.sh -sSf | sh
sh: 13: Syntax error: "(" unexpected (expecting "}")
```

Line 13:
```
  rc_files=("$HOME/.bashrc" "$HOME/.zshrc")
```

I am just tried replace the array like below, removing the bracket indeed works, both for `bash` and `sh`.
Can't say what the issue tho.
```
  rc_files="$HOME/.bashrc" "$HOME/.zshrc"
```